docs/theme/mlflow/static/js/custom.js

```diff
@@ -298,59 +227,3 @@ fetch('https://pypi.org/pypi/mlflow/json')
   .catch((error) => {
     console.error('Failed to fetch package metadata from PyPI:', error);
   });
-
-function addCopyButton()  {
```

Comment:
When enabled, this copy button was applied to all cells (code, markdown, and results cells) both :before: and :after: each element. 
The functionality was replaced in the first docs overhaul, so this is dead code.